registry/tracecat_registry/integrations/ldap3.py

```diff
@@ -195,3 +211,65 @@ def enable_active_directory_user(
     with create_ldap_client(use_ssl=use_ssl, is_active_directory=True) as client:
         result = client.enable_user(user_dn)
         return result
+
+@registry.register(
+    default_title="Expire AD/LDAP User",
+    description="Expire AD or LDAP user by distinguished name",
+    display_group="LDAP",
+    namespace="integrations.ldap",
+    secrets=[ldap_secret],
+)
+def expire_active_directory_user(
+    user_dn: Annotated[
+        str,
+        Field(..., description="User distinguished name"),
+    ],
+    expiration_datetime: Annotated[
+        Union[datetime, int],
+        Field(..., description="Datetime for expiration : Defaults to NOW, set to '0' to unexpire"),
```

Comment:
```suggestion
        Field(..., description="Expiration datetime. If None, defaults to UTC now."),
```

Reply:
Has the set to 0 logic been tested? I think it's better to have a separate action entirely to "unexpire". Thoughts?